MAUI/Autocomplete/UI-Customization.md

```diff
@@ -1240,3 +1240,80 @@ sfAutocomplete.ReturnType = ReturnType.Next;
 {% endtabs %}
 
 ![.NET MAUI Autocomplete ReturnType](Images/UICustomization/ReturnType.png)
+
+## Clear button customization
+
+The `ClearButtonPath` property allows users to set the path for customizing the appearance of the [SfAutocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) clear Button.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfAutocomplete x:Name="autoComplete"
+                    ItemsSource="{Binding SocialMedias}"
+                    TextMemberPath="Name"
+                    DisplayMemberPath="Name"
+                    HeightRequest="40"
+                    WidthRequest="240">
+            <editors:SfAutocomplete.ClearButtonPath>
+                <Path Data="M1.70711 0.292893C1.31658 -0.097631 0.683417 -0.097631 0.292893 0.292893C-0.097631 0.683417 -0.097631 1.31658 0.292893 1.70711L5.58579 7L0.292893 12.2929C-0.097631 12.6834 -0.097631 13.3166 0.292893 13.7071C0.683417 14.0976 1.31658 14.0976 1.70711 13.7071L7 8.41421L12.2929 13.7071C12.6834 14.0976 13.3166 14.0976 13.7071 13.7071C14.0976 13.3166 14.0976 12.6834 13.7071 12.2929L8.41421 7L13.7071 1.70711C14.0976 1.31658 14.0976 0.683417 13.7071 0.292893C13.3166 -0.097631 12.6834 -0.097631 12.2929 0.292893L7 5.58579L1.70711 0.292893Z" 
+                Fill="Red" 
+                Stroke="Red"/>
+            </editors:SfAutocomplete.ClearButtonPath>
+</editors:SfAutocomplete>
+
+{% endhighlight %}
+{% highlight c# %}
+
+private string _customPath = "M1.70711 0.292893C1.31658 -0.097631 0.683417 -0.097631 0.292893 0.292893C-0.097631 0.683417 -0.097631 1.31658 0.292893 1.70711L5.58579 7L0.292893 12.2929C-0.097631 12.6834 -0.097631 13.3166 0.292893 13.7071C0.683417 14.0976 1.31658 14.0976 1.70711 13.7071L7 8.41421L12.2929 13.7071C12.6834 14.0976 13.3166 14.0976 13.7071 13.7071C14.0976 13.3166 14.0976 12.6834 13.7071 12.2929L8.41421 7L13.7071 1.70711C14.0976 1.31658 14.0976 0.683417 13.7071 0.292893C13.3166 -0.097631 12.6834 -0.097631 12.2929 0.292893L7 5.58579L1.70711 0.292893Z";
+
+var converter = new PathGeometryConverter();
+var path = new Path() 
+{ 
+    Data = (PathGeometry)converter.ConvertFromInvariantString(_customPath;),
+    Fill = Colors.Red,
+    Stroke = Colors.Red
+};
+
+autoComplete.ClearButtonPath = path;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete ClearButtonPath](Images/UICustomization/clearbuttoncustomization.png)
+
+## Command
+
+### Return Command and Return Command Parameter
+
+`ReturnCommand` and `ReturnCommandParameter` allows users to customize and handle action invoked by pressing the Enter key.
+
+- `ReturnCommand` : Specifies what should happent when a [SfAutocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html)control is interacted (e.g., clicking a button or pressing Enter key in an entry field).
```

Comment:
ReturnCommand, of type [ICommand](https://learn.microsoft.com/en-us/dotnet/api/system.windows.input.icommand), defines the command to be executed when the return key is pressed.

Reply:
content updated